model/canvas.cc

```diff
@@ -167,7 +167,27 @@ namespace minsky
       case LassoMode::itemResize:
         if (item)
           {
-            item->resize(lasso);
+            // resize both operator and variable in coupled integral variable. for feature 94
+            if (auto i=dynamic_cast<IntOp*>(item.get())) {
+               if (i->coupled()) {
+				  i->resize(lasso); 
```

Comment:
This line is common to all branches of the conditional, so should be moved before the first if statement.

Reply:
I fixed this by providing IntOp with its own draw() and drawResizeHandles() methods. So, now only item->resize(lasso).